index.html

```diff
@@ -56,21 +56,26 @@
                 <ul class="nav navbar-nav navbar-right">
 
                     <!-- navigation menu -->
-		            <li class="active"><a data-scroll href="index.html">Attend</a></li>
+                    <li class="active"><a data-scroll href="index.html">Attend</a></li>
                     <li><a data-scroll href="#about" title="About SMConf 2016- Silicon Mountain Conference">About</a></li>
                     <li><a data-scroll href="#speakers" title="Speakers for SMConf 2016 - Silicon Mountain Conference">Speakers</a></li>              
                     <li><a data-scroll href="#schedule" title="Schedule SMConf 2016 - Silicon Mountain Conference">Schedule</a></li>                  
                     <li><a data-scroll href="#partner" title="Partners of SMConf 2016 - Silicon Mountain Conference">Partners</a></li>                  
                     <li><a data-scroll href="#sponsor" title="Sponsors for SMConf 2016 - Silicon Mountain Conference">Sponsors</a></li>
                     <li><a data-scroll href="#faq" title="Questions about SMConf 2016 - Silicon Mountain Conference">FAQ</a></li>
-                    <li><a data-scroll href="#photos" title="Photos of SMConf 2015 - Silicon Mountain Conference">2015</a></li>
-		            <li><a data-scroll href="#location" title="Chariot Hotel Buea, the Venue of SMConf 2016 - Silicon Mountain Conference">Venue</a></li>
-                
+                    <li id="pst"><a data-scroll href="#" title="Photos of SMConf 2015 - Silicon Mountain Conference">Past Events</a>
+                    <ul id="dropdown">
+                        <li> <a href="#"> 2016 </a></li>
+                        <li> <a href="#photos"> 2015 </a></li>
+                        <li> <a href="#"> 2014 </a></li>
+                        <li> <a href="#"> 2013 </a></li>
```

Comment:
The Maiden ( First ) Silicon Mountain Conference was SmConf 2015 which took place on June 20th in Eta Palace Hotel.